fern/pages/docs/getting-started/global-configuration.mdx

```diff
@@ -409,4 +409,7 @@ landing-page:
 
 <ParamField path="path" type="string" required={true}>
   Relative filepath to the desired landing page Markdown file. 
-</ParamField>
\ No newline at end of file
+</ParamField>
+
+See VapiAI's landing page live [here](https://docs.vapi.ai/welcome) and the associated Markdown file (`welcome.mdx`) [here](https://github.com/VapiAI/docs/blob/main/fern/welcome.mdx?plain=1).
```

Comment:
Small nit: for accessibility, it's best practice to link directly from the text that describes what you're linking to. 

Ex: `See [VapiAI's landing page live](link_here) and the associated [Markdown file](link_here)`